utils/dummy_nodes.py

```diff
@@ -76,7 +76,7 @@ def create_dummy_class(bl_id):
         {
             'bl_idname': bl_id,
             'bl_label': label,
-            "missing_dependecy": dependency_name,
+            "missing_dependency": dependency_name,
```

Comment:
@vicdoval What do you think, can this break backward compatibility?

Reply:
At a first glance I think it wont break anything. Maybe if you save a nodetree with a dummy node and you try to open it again it may complain, but it will be a ui complain